products/health-care/appointments/va-online-scheduling/express-care-research/express-care-project-brief.md

```diff
@@ -1,76 +1,71 @@
 
-## Problem Statement:
-Veterans have medical needs that may either be resolved without an in-person appointment and/or want guidance about how seek additional care for a medical concern, yet may not demand a trip to the ER. For various reasons, the Veteran may not be able to go into a physical location for this care. In both cases, however, the Veteran needs to talk to a medical professional *today*.
+As a Veteran who lives in a geographical COVID-19 hot spot, I need to be provided a way get immediate COVID-19 screening and evaluation from my VA doctor/PACT team. 
 
-Currently, "Express Care" is a care option developed by VA for immediate, same day evaluation via VA Video Connect (VA's proprietary telehealth platform). However, Express Care is not utilized by VA providers or patients and we believe the business rules in place prevent this platform from being a preferred option by providers and patients, alike.
+As a VA facility located in a COVID-19 hot spot, I need to be able to screen and evaluate VA patients through a digital medium so that I can limit exposure of VA staff and patients to potential COVID-19 cases and/or alleviate need for in-person evaluation this is creating operational bottlenecks. I want to do this by PACT.
+
+### Out of scope problem statements
 
-This problem may also be a subset to a larger problem statement - Veterans have access to various types of care which can make it difficult to understand what kind of care to access and how to access it. This is especially true when considering the different VA digital access points to accessing care.
+As the VA, the business rules created that dictate availability of Express Care scheduling prevent Veterans from utilizing  this platform as a viable scheduling option.
 
-Additionally, with the recent outbreak of the COVID-19 pandemic, there is an urgent need to give Veterans fast, safe access to care and evaluation via telehealth either as an alternative to in office, face-to-face visits. The VAOS team believes Express Care can serve as a tool to facilitate immediate access to remote medical care as well as provide VA providers a remote mechanism for screening patients who suspect they have COVID-19.
+As a VA medical provider, the business rules that dictate scheduling for Express Care prevent me from using the platform as a way to treat my patients. 
 
 ## Goal:
 We want to perform usability research with Veterans to learn more about:
- + Veterans undertanding of what appointment types are available to them through VA, specifically for telemedicine and same-day appointments
-  + how might Veterans seek same-day care (either through VA or other sources)
-  + how might Veterans go about receiving immediate care at the VA, with emphasis on understanding use of 'telehealth' or online care options. 
+- Veterans understanding of what appointment types are available to them through VA, specifically for telemedicine and same-day appointments
```

Comment:
I think the phrasing "types" will be confusing to Veterans. type of care = specialty; "different WAYS (modalities) to receive care" = in person/phone/video, etc.